### DISCUSSION TITLE
New dates for our open Weekly Calls

### DISCUSSION THREAD

**Discussion by santisoler** [fatiando/community#112] [Announcements]
**We have new dates and times for our Weekly Fatiando Development Calls!** Starting on October 12th, 2023, the calls will alternate weekly between Thursday at 17:00 UTC and Thursday at 00:00 UTC.
In summary, we moved the previous time from Wednesday 20:00 UTC to Thursday 17:00 UTC. Calls on Wednesday have been experiencing very low attendance in the last months, so we decided to move it to a time that would be more suitable for people.

📱 Join the video call: https://meet.jit.si/fatiando-a-terra
📆 Shared Google calendar with meeting times: https://www.fatiando.org/calendar

**Everyone is welcome!** 

---

* ℹ️ More information: https://github.com/fatiando/community
* 💬 Join our chat rooms to get notified of calls: https://www.fatiando.org/contact/
* 🗒️ Meeting notes discussing this change: https://hackmd.io/@fatiando/development-calls-2023#2023-10-5.

**Comment by MGomezN** [comment]
I have updated this post according to the current meeting times :)

**Comment by MGomezN** [comment]
I'm closing this discussion, to keep open the announcement with the most up to date meeting times :)
